### PR TITLE
Add postprocess pipeline

### LINKED ISSUE #117: Postprocessing pipeline for DBpedia tags
Create a post-processing pipeline for the DBpedia tags extracted from the four datasets:

- [ ] Remove place names, people etc. using NER
- [ ] Output data in a useable format

### PR DESCRIPTION
**Pull request by Jack-Vines**
Closes #117 

Adds a flow to postprocess all generated lookups, and store results in the same input folder (with `_clean` added to filename).

---

Checklist:

- [x] I have refactored my code out from `notebooks/`
- [x] I have checked the code runs
- [x] I have tested the code
- [x] I have run `pre-commit` and addressed any issues not automatically fixed
- [x] I have merged any new changes from `dev`
- [x] I have documented the code
  - [x] Major functions have docstrings
  - [x] Appropriate information has been added to `README`s
- [x] I have explained the feature in this PR or (better) in `output/reports/`
- [x] I have requested a code review

### CODE REVIEW

File: ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+from metaflow import FlowSpec, step, retry
+
+import ast
+import boto3
+import json
+
+s3 = boto3.resource("s3")
+client = boto3.client("s3")
+path = "outputs/entity_extraction/"
+bucket = "ai-genomics"
```
Comment (india-kerle) [inline ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py]: you can import this from ai_genomics i.e.:

`from ai_genomics import bucket_name`

File: ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+from ai_genomics.pipeline.entity_extraction.postprocess_entities import EntityCleaner
+from ai_genomics.getters.data_getters import save_to_s3
+
+from metaflow import FlowSpec, step, retry
+
+import ast
+import boto3
+import json
+
+s3 = boto3.resource("s3")
```
Comment (india-kerle) [inline ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py]: you can also import this from ai_genomics.getters.data_getters, i.e:

`from ai_genomics.getters.data_getters import S3`

File: ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    def join(self, inputs):
+        self.next(self.end)
+
+    @step
+    def end(self):
+        pass
+
+
+if __name__ == "__main__":
+    PostprocessEntitiesFlow()
```
Comment (india-kerle) [inline ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py:62]: nice - I've checked the output of the `_clean.jsons` and it looks as expected.

**Review by india-kerle — COMMENTED** [review]
looks good jack - just some minor comments on importing the bucket_name and s3.

**Comment by Jack-Vines** [comment]
Thanks @india-kerle, i've updated based on your comments and added getters for each of the files :)

**Comment by india-kerle** [comment]
@Jack-Vines gorgeous!! thanks for the getters also - feel free to merge.

**Pull request merged**